cmake/Modules/OpmSatellites.cmake

```diff
@@ -266,6 +267,11 @@ macro(opm_add_test TestName)
   # case. They can still be build using 'make test-suite' and they can
   # be build and run using 'make check'
   set(CURTEST_EXCLUDE_FROM_ALL "")
+  if (NOT "AND OR ${CURTEST_DEFAULT_ENABLE_IF}" STREQUAL "AND OR ")
```

Comment:
why not just if NOT "${CURTEST_DEFAULT_ENABLE_IF}" STREQUAL "" ? apart from that it looks fine (this will work as well but looks like a bash hack).

Reply:
I tried this while doing this patch, but it did not work. This is probably because cmake seems to have trouble with distinguishing a string from a variable name, i.e., depending on what the CURTEST_DEFAULT_ENABLE_IF variable contains, cmake interprets the *contents* as a variable name which obviously is almost always empty. Also, for some reason `if (CURTEST_DEFAULT_ENABLE_IF STREQUAL "")` did not work either. tl;dr: STREQUAL in cmake is sketchy...